### PR TITLE
Adds the M16 and compatible magazines to req vendors

### PR DESCRIPTION
**Pull request by Gooboi**
Adds 5 M16s to req gun vendors, and 50 M16 magazines to req ammo vendors.

## About The Pull Request

Adds 5 of the FN M16A4 rifles to the requisitions armaments vendors, and 50 compatible magazines to requisitions munitions vendors.

## Why It's Good For The Game

The M16 can be a fun alternative to the T-12; 20 less rounds per magazine, and 20% scatter, for a 20% damage modifier.

## Changelog
:cl:
add: The FN M16A4 can now be found in requisitions vendors!
/:cl:

### CODE REVIEW

**Comment by Gooboi** [comment]
Under all the other standard rifle type weapons you mean? Or every standard weapon?

**Comment by psykzz** [comment]
In the rifle section but under the standard rifles

**Comment by psykzz** [comment]
LGTM. I'd like to hear back from some balance guys about giving this away in the vendor for free though.

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -108,13 +108,13 @@
...
 					/obj/item/weapon/gun/smg/standard_smg = 10,
 					/obj/item/weapon/gun/smg/standard_machinepistol = 10,
-					/obj/item/weapon/gun/rifle/m16 = 5,
 					/obj/item/weapon/gun/rifle/standard_carbine = 10,
 					/obj/item/weapon/gun/rifle/standard_assaultrifle = 10,
 					/obj/item/weapon/gun/rifle/standard_lmg = 10,
 					/obj/item/weapon/gun/rifle/standard_gpmg = 10,
 					/obj/item/weapon/gun/rifle/standard_dmr = 10,
 					/obj/item/weapon/gun/rifle/standard_br = 10,
+					/obj/item/weapon/gun/rifle/m16 = 5,
```
Comment (Gooboi) [inline code/game/objects/machinery/vending/marine_vending.dm:117]: moved the M16 under the standard rifles

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 2]
```diff
@@ -167,13 +167,13 @@
...
 					/obj/item/ammo_magazine/smg/standard_smg = 50,
 					/obj/item/ammo_magazine/smg/standard_machinepistol = 50,
-					/obj/item/ammo_magazine/rifle/m16 = 50,
 					/obj/item/ammo_magazine/rifle/standard_carbine = 50,
 					/obj/item/ammo_magazine/rifle/standard_assaultrifle = 50,
 					/obj/item/ammo_magazine/standard_lmg = 50,
 					/obj/item/ammo_magazine/standard_gpmg = 50,
 					/obj/item/ammo_magazine/rifle/standard_dmr = 50,
 					/obj/item/ammo_magazine/rifle/standard_br = 50,
+					/obj/item/ammo_magazine/rifle/m16 = 50,
```
Comment (Gooboi) [inline code/game/objects/machinery/vending/marine_vending.dm:176]: moved M16 ammo under standard rifle ammo

**Comment by Chaznoodles** [comment]
No.

We had a period a while back when someone TM'd reducing points values of surplus things in Req to 5 - this was one of them. If anyone remembers this, they'll note how absolutely ridiculous all of the surplus guns - including the M16 - are when easily accessed. They should not be freely available at round start. To note: This is ten M16s and 100 magazines on the Sulaco.

(Not to mention us already having so many standard-issue guns that we don't need yet more clogging up the vendors that serve exactly the same function)

**Pull request closed without merging**